### PR TITLE
admin: add new command for creating KMS master keys

### PR DESCRIPTION
**Pull request by aead**
This commit adds a new admin command for creating
KMS master keys:
```
mc admin kms key create <alias> <key-name>
```

This command requires a MinIO server with the
create KMS key admin API added by minio/minio#9982

### CODE REVIEW

File: cmd/admin-kms-key-create.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	"github.com/fatih/color"
+	"github.com/minio/cli"
+	"github.com/minio/mc/pkg/probe"
+	"github.com/minio/minio/pkg/console"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+var adminKMSCreateKeyCmd = cli.Command{
+	Name:   "create",
+	Usage:  "creates a new master key at the KMS",
```
Comment (harshavardhana) [inline cmd/admin-kms-key-create.go:32]:
```suggestion
	Usage:  "create a new master key on KMS",
```

File: cmd/admin-kms-key-create.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+// adminKMSCreateKeyCmd is the handler for the "mc admin kms key create" command.
+func mainAdminKMSCreateKey(ctx *cli.Context) error {
+	if len(ctx.Args()) != 2 {
+		cli.ShowCommandHelpAndExit(ctx, "create", 1) // last argument is exit code
+	}
+
+	client, err := newAdminClient(ctx.Args().Get(0))
+	fatalIf(err, "Cannot get a configured admin connection.")
+
+	keyID := ctx.Args().Get(1)
```
Comment (harshavardhana) [inline cmd/admin-kms-key-create.go:60]: is there a restriction on what kind of name it must be any length?

File: cmd/admin-kms-key-create.go [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+	Name:   "create",
+	Usage:  "creates a new master key at the KMS",
+	Action: mainAdminKMSCreateKey,
+	Before: initBeforeRunningCmd,
+	Flags:  globalFlags,
+	CustomHelpTemplate: `NAME:
+  {{.HelpName}} - {{.Usage}}
+
+USAGE:
+  {{.HelpName}} TARGET [KEY_NAME]
```
Comment (harshavardhana) [inline cmd/admin-kms-key-create.go:40]: [KEY_NAME] indicates that its optional, do we autogenerate key if not provided? if not it should be 

```suggestion
  {{.HelpName}} TARGET KEYNAME
```

File: cmd/admin-kms-key-create.go (hunk 2 above)
Comment (aead) [reply cmd/admin-kms-key-create.go:60]: Not really. It must be a valid URL path segment (e.g. not contain `/`). There isn't a explicit size restriction but Go must be able to create an URL. So I guess it cannot be larger then a couple of MiB which is anyway odd.
In any case such a restriction should be enforced at the server-side.

**Comment by harshavardhana** [comment]
@aead is this PR ready can this be rebased with any new changes needed?

**Review by harshavardhana — APPROVED** [review]

**Comment by harshavardhana** [comment]
minio/minio#b8cb21c954

**Pull request merged**